RUFAS/routines/animal/ration/ration_driver.py

```diff
@@ -287,12 +287,12 @@ def make_ration_from_solution(
         """
         ration: Dict[str, float | str] = {}
         for feed_id in range(len(available_feeds["feed_id"])):
-            i = feed_id * 3
-            num = solution.x[i] + solution.x[i + 1] + solution.x[i + 2]
+            i = feed_id
+            num = solution.x[i]
```

Comment:
```suggestion
            num = solution.x[feed_id]
```
Maybe this is just because I don't have a good understanding of what is going on in Ration Formulation, but maybe there is a more descriptive name for this variable than "num"

Reply:
Great suggestion here, and I think changing `num` and `i` will help demystify this part of the codebase.